src/components/InquiryRows.jsx

```diff
@@ -66,6 +81,38 @@ const InquiryRows = ({ item }) => {
                     </p>
                   </>
                 )}
+                <p>
+                  Status:
+                  <span data-cy='inquiry-status'>{item.inquiry_status}</span>
+                </p>
+                <div className='status-buttons-container'>
+                  <FormControl component='fieldset'>
+                    <RadioGroup
+                      name='status-action'
+                      value={inquiryStatus}
+                      onChange={(event) => statusHandler(event.target.value)}>
+                      <FormControlLabel
+                        data-cy='status-btn-1'
+                        value='pending'
+                        control={<Radio />}
+                        label='Pending'
+                      />
+                      <FormControlLabel
+                        data-cy='status-btn-2'
+                        value='started'
+                        control={<Radio />}
+                        label='Started'
+                      />
+                      <FormControlLabel
+                        data-cy='status-btn-3'
+                        value='done'
+                        control={<Radio />}
+                        label='Done'
+                      />
+                    </RadioGroup>
+                  </FormControl>
+                </div>
+                {error && <ErrorMessage text={message} />}
```

Comment:
Material UI has this component that can be cool for errors. https://material-ui.com/components/snackbars/
So the table can say something like 'No inquiries yet' but the actual 500 error txt can be in snackbar on top of the screen